### PR TITLE
ice: blank out ufrag on Candidate once added

### PR DESCRIPTION
**Pull request by algesten**
- ice: blank out ufrag on Candidate once added
- Clippy fixes

### CODE REVIEW

File: src/ice/agent.rs [hunk 1]
```diff
@@ -365,6 +367,9 @@ impl IceAgent {
             }
         }
 
+        // "Adopt" any incoming candidate by setting our current ufrag.
+        c.set_ufrag(&self.local_credentials.ufrag);
```
Comment (k0nserv) [inline src/ice/agent.rs:371]: How was this getting set before?

Comment (algesten) [reply src/ice/agent.rs:371]: Not sure it was other than parsed from the SDP.

Comment (k0nserv) [reply src/ice/agent.rs:371]: From the remote? But it returns its won clients, not ours

**Review by k0nserv — APPROVED** [review]
Let's do it

**Pull request merged**